tests/page-route.spec.ts

```diff
@@ -536,8 +536,6 @@ it('should support cors with GET', async ({page, server}) => {
 });
 
 it('should support cors with POST', async ({page, server, isChromium, browserChannel, isElectron, isAndroid}) => {
```

Comment:
nit: can you please remove `isChromium`, `browserChannel`, `isElectron` and `isAndroid`?

Reply:
done!

I looked into changing the eslint config to enforce this, but it would be a pretty big change across the code base, both tests and implementation.

```
'@typescript-eslint/no-unused-vars': [2, {args: 'all', argsIgnorePattern: '^_'}]
```

Even if I just scope it to the `tests` directory, there's quite a few instances of:

```
(req, res) => { … }
```

where one of the arguments is unused, but definitely the majority of the violations are `server`/fixture references in the tests.

Reply:
That being said, if we want the above config added, I'm happy to add it (and do the therapeutic work) of removing unused vars from our tests and `_prefix`ing where necessary within the tests.